user.js

```diff
@@ -601,7 +601,7 @@ user_pref("network.IDN_show_punycode", true);
  * [SETUP-CHROME] You may prefer a different pdf reader for security/workflow reasons
  * [SETTING] General > Files & Attachments > Portable Document Format (PDF)
  * [1] https://cve.mitre.org/cgi-bin/cvekey.cgi?keyword=pdf.js+firefox ***/
-user_pref("pdfjs.disabled", false); // [DEFAULT: false]
+// user_pref("pdfjs.disabled", false); // [DEFAULT: false]
```

Comment:
The rationale here was : PDFJS is likely more up to date (and with a smaller attack surface) than system PDF readers/editors (even more on Windows).
Moreover, I guess it's better from a security point of view if files coming from e-mails stay handled by the e-mail client.

(note : I would agree recent CVE #53 does not score points here, but I _guess_ it's better overall, if we compare to [Adobe's Acrobat](https://helpx.adobe.com/security/security-bulletin.html#acrobat), for instance)

Reply:
but PDF JS is enabled by default and will forever be as it is a supported feature. So I think this is not needed, unless Adobe malware would change that setting.

Reply:
I think we'll keep this one enforced for enterprise environments that _may_ ship system-wide configuration disabling pdfjs, prefering an installed licensed software or whatever.